src/lib/Tab/URLManagement/URLWildcardsTab.php

```diff
@@ -97,42 +107,99 @@ public function getOrder(): int
      */
     public function renderView(array $parameters): string
     {
-        $currentPage = $this->requestStack->getCurrentRequest()->query->getInt(
+        $limit = $this->configResolver->getParameter('pagination.url_wildcards');
+        $currentRequest = $this->requestStack->getCurrentRequest();
+        $page = $this->requestStack->getCurrentRequest()->query->getInt(
             self::PAGINATION_PARAM_NAME, 1
         );
-        $limit = $this->configResolver->getParameter('pagination.url_wildcards');
 
-        $pagerfanta = new Pagerfanta(
+        $data = new URLWildcardListData();
+        $data->setLimit($limit);
+
+        $searchUrlWildcardForm = $this->formFactory->create(
+            URLWildcardListType::class,
+            $data,
+            [
+                'method' => Request::METHOD_GET,
+                'csrf_protection' => false,
+            ]
+        );
+
+        $searchUrlWildcardForm->handleRequest($currentRequest);
+
+        if ($searchUrlWildcardForm->isSubmitted() && !$searchUrlWildcardForm->isValid()) {
+            throw new BadRequestHttpException();
+        }
+
+        $urlWildcardLists = new Pagerfanta(
             new URLWildcardAdapter(
+                $this->buildListQuery($data),
                 $this->urlWildcardService
             )
         );
-        $pagerfanta->setMaxPerPage($limit);
-        $pagerfanta->setCurrentPage(min(max($currentPage, 1), $pagerfanta->getNbPages()));
 
-        $urlWildcards = $pagerfanta->getCurrentPageResults();
+        $urlWildcardLists->setMaxPerPage($data->getLimit());
+        $urlWildcardLists->setCurrentPage(min($page, $urlWildcardLists->getNbPages()));
+
+        $urlWildcards = $urlWildcardLists->getCurrentPageResults();
         $urlWildcardsChoices = [];
         foreach ($urlWildcards as $urlWildcardItem) {
             $urlWildcardsChoices[$urlWildcardItem->id] = false;
         }
 
-        $deleteUrlWildcardDeleteForm = $this->formFactory->deleteURLWildcard(
+        $deleteUrlWildcardDeleteForm = $this->formFactory->create(
+            URLWildcardDeleteType::class,
             new URLWildcardDeleteData($urlWildcardsChoices)
         );
 
-        $addUrlWildcardForm = $this->formFactory->createURLWildcard();
+        $addUrlWildcardForm = $this->formFactory->create(
+            URLWildcardType::class,
+            new URLWildcardData()
```

Comment:
Not needed because of: https://github.com/ezsystems/ezplatform-admin-ui/blob/2.3/src/lib/Form/Type/URLWildcard/URLWildcardType.php#L48